### PR TITLE
🔥 Strip taco api down to minimum necessary

### PR DESCRIPTION
**Pull request by meatballs**

### CODE REVIEW

**Comment by meatballs** [comment]
I've realised I can take this further. I'll do that at some point soon.

**Comment by meatballs** [comment]
all done and ready for review!

**Comment by meatballs** [comment]
A question... I'm calling `_taco.initialize()` in the taco module. Is that ok or should I just export that function like anything else and let the user choose when to call it?

**Comment by meatballs** [comment]
I think I'm minded to export it and leave it to the user

**Comment by meatballs** [comment]
All yours @theref . There's not much more that could be stripped down now!

File: .anvil_editor.yaml
Comment (theref) [inline .anvil_editor.yaml:1]: this shouldn't be included

**Review by theref — APPROVED** [review]

**Pull request merged**